mmpose/codecs/wrappers.py

```diff
@@ -1,99 +0,0 @@
-# Copyright (c) OpenMMLab. All rights reserved.
-from typing import Any, List, Optional, Tuple
-
-import numpy as np
-
-from mmpose.registry import KEYPOINT_CODECS
-from mmpose.utils.typing import ConfigType
-from .base import BaseKeypointCodec
-
-
-@KEYPOINT_CODECS.register_module()
-class MultiLevelHeatmapEncoder(BaseKeypointCodec):
-    """An encoder wrapper that contain multiple encoders to generate multi-
-    level heatmaps.
-
-    Note:
-
-        - encoder number: n
-        - instance number: N
-        - keypoint number: K
-        - keypoint dimension: D
-        - image size: [w, h]
-        - heatmap size: [W, H]
-
-    Args:
-        encoders (List[dict]): The encoder config list that should contain
-            at least two encoder configs
-    """
-
-    def __init__(self, encoders: List[ConfigType]):
-        super().__init__()
-
-        assert isinstance(encoders, list) and len(encoders) > 1, (
-            '``encoders`` should be a list of more than 1 configs')
-
-        self.encoders = [KEYPOINT_CODECS.build(cfg) for cfg in encoders]
-
-        self.input_size = self.encoders[0].input_size
-        self.heatmap_size = self.encoders[0].heatmap_size
-
-        # check all encoders have the samle ``input_size`` and
```

Comment:
typo: samle - same

Reply:
This file is deleted. Multi-scale heatmaps will be directly generated by the data transform `GenerateTarget`